config/element-api.php

```diff
@@ -455,8 +455,7 @@ function getPublication($locale, $programmeSlug, $pageSlug = null)
         'criteria' => $criteria,
         'meta' => $meta ?? null,
         'transformer' => function (Entry $entry) use ($locale) {
-            list('entry' => $entry, 'status' => $status) = EntryHelpers::getDraftOrVersionOfEntry($entry);
-            $common = ContentHelpers::getCommonFields($entry, $status, $locale);
+            $common = ContentHelpers::getCommonFields($entry, $entry->status, $locale);
```

Comment:
Because we are already passing `$entry` to `getCommonFields` we could refactor this method to no longer accept the status argument and pull the value from the entry directly.